portfolio/src/main/webapp/comments.js

```diff
@@ -0,0 +1,82 @@
+
+function getUserAvatar() {
+  const avatar = document.createElement("a");
+  avatar.className = "pull-left";
+  avatar.setAttribute("href", "http://google.com");
+
+  function getAvatarImage() {
```

Comment:
This function surprises me, why does it need to be nested?

Reply:
It was nested because the function returns returns a image with a specific class name. I modified it and now it's more generic.